### ISSUE TITLE
[template] Proposed Recommendation and Success Criteria

### ISSUE THREAD

**Issue by plehegar** [w3c/charter-drafts#485]
[[
In order to advance to [Proposed Recommendation](https://www.w3.org/Consortium/Process/#RecsPR), each normative specification is expected to have [at least two independent interoperable implementations](https://www.w3.org/Consortium/Process/#implementation-experience) of every feature defined in the specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperating with each other. In order to advance to Proposed Recommendation, each normative specification must have an open test suite of every feature defined in the specification.
]]
https://w3c.github.io/charter-drafts/charter-template.html

This could be simplified, since the last sentence seems to repeat the first sentence.

**Comment by svgeesus** [comment]
Agreed. I think the second sentence is the original one, and the longer first sentence was added to clarify it.

**Comment by tidoust** [comment]
Reopening following discussion in https://github.com/w3c/charter-media-wg/pull/53 that re-introduces the sentence in the Media WG charter, noting @tantek's comment in particular:

> The short version is that the first sentence has a comprehensive implementation requirement for every feature, but not specifically that each feature itself is tested in a test suite, only that there is an open test suite.
> 
> The second sentence makes it crystal clear that literally every feature must have tests.
> 
> It's not a duplicate.
> 
> Quite the opposite, it's necessary to clearly state both requirements, to both inform WG decision-making, staff contacts, and the electorate voting for potential RECs, potentially FOs based on violating this requirement, and any FO council that would be formed to process such an FO.

**Comment by plehegar** [comment]
Fixed in https://github.com/w3c/charter-drafts/pull/684
